File: selector/src/main/java/com/luck/picture/lib/interfaces/OnResultCallbackListener.java
package com.luck.picture.lib.interfaces;

import java.util.ArrayList;

/**
 * @author：luck
 * @date：2020-01-14 17:08
 * @describe：onResult Callback Listener
 */
public interface OnResultCallbackListener<T> {
    /**
     * return LocalMedia result
     *
     * @param result
     */
    void onResult(ArrayList<T> result);

    /**
     * Cancel
     */
    void onCancel();
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnResultCallbackListener.kt
package com.luck.picture.lib.interfaces

import com.luck.picture.lib.entity.LocalMedia

/**
 * @author：luck
 * @date：2020-01-14 17:08
 * @describe：onResult Callback Listener
 */
interface OnResultCallbackListener {
    /**
     * return LocalMedia result
     *
     * @param result
     */
    fun onResult(result: List<LocalMedia>)

    /**
     * Cancel
     */
    fun onCancel()
}